.rubocop.yml

```diff
@@ -0,0 +1,37 @@
+require: 
+  - rubocop-rails
+
+AllCops:
+  NewCops: enable
+  Exclude:
+    - 'db/**/*'
+    - 'bin/**/*'
+    - 'Gemfile'
+    - 'Rakefile'
+    - 'spec/spec_helper.rb'
+    - 'node_modules/**/*'
+    - 'tmp/*'
+    - 'config/initializers/**/*'
+    - 'lib/tasks/*'
+    - 'Guardfile'
+
+Layout/LineLength:
+  Max: 100
+  Exclude: 
+    - 'config/routes.rb'
```

Comment:
I'd love to keep `routes.rb` in the fold. Looks like there's only one line currently exceeding `100`.

Reply:
sounds good to me